less/button.less

```diff
@@ -82,6 +82,7 @@ span.buttonSeparator {
     padding-bottom: 5px;
     width: auto;
     height: auto;
+    min-width: calc(@defaultFontSize * 6); // issue #6384
```

Comment:
As far as I know via the picture on https://github.com/brave/browser-laptop/issues/6384#issue-197094544 where you see primaryButton, .actionButton, .wideButton, .subtleButton, and .whiteButton, I don't think so.